app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -135,7 +136,9 @@ class BrowserFragment : IWebViewLifecycleFragment(),
                 reload()
             }
             NavigationEvent.PIN_SITE -> {
-                // TODO: Add to pinned sites
+                if (webview.getUrl() != null) {
```

Comment:
You should use `BrowserFragment.url` – we set it outside of the webView so we have more control over it. THis is a hold-over from Focus that I don't entirely understand why.